src/style.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.bigSearchBar {
+  padding: 27px 0;
+  margin: 12px 0;
+  padding-left: 61px;
+  font-size: 16px;
+}
```

Comment:
these styles to .search, delete styles for .bigSeachBar, leave styles for .smallSearchBar
(checklist point 2)